libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/model/NavigationRouteLine.kt

```diff
@@ -0,0 +1,26 @@
+package com.mapbox.navigation.ui.maps.route.line.model
+
+import com.mapbox.navigation.base.route.NavigationRoute
+import com.mapbox.navigation.base.route.toNavigationRoute
+
+/**
+ * Represents a route and an optional identification of used for representing routes on the map.
+ *
+ * @param route a directions route
+ * @param identifier an optional identifier for the directions route which can be used to
+ * influence color of the route when it is an alternative route.
+ */
+data class NavigationRouteLine(val route: NavigationRoute, val identifier: String?)
+
+/**
+ * Maps [RouteLine] to [NavigationRouteLine].
+ */
+fun RouteLine.toNavigationRouteLine() = NavigationRouteLine(
+    route.toNavigationRoute(),
+    identifier
+)
+
+/**
+ * Maps [RouteLine]s to [NavigationRouteLine]s.
+ */
+fun List<RouteLine>.toNavigationRouteLines() = map { it.toNavigationRouteLine() }
```

Comment:
We should probably move these extensions to a different file `@file:JvmName("NavigationRouteLineEx")`. Refs. https://github.com/mapbox/mapbox-navigation-android/pull/5411#discussion_r793700629

Reply:
I added the `file:JvmName` name to the existing file and it's generating correctly:
```
  public final class NavigationRouteEx {
    method public static java.util.List<com.mapbox.api.directions.v5.models.DirectionsRoute> toDirectionsRoutes(java.util.List<com.mapbox.navigation.base.route.NavigationRoute>);
    method public static com.mapbox.navigation.base.route.NavigationRoute toNavigationRoute(com.mapbox.api.directions.v5.models.DirectionsRoute);
    method public static java.util.List<com.mapbox.navigation.base.route.NavigationRoute> toNavigationRoutes(java.util.List<? extends com.mapbox.api.directions.v5.models.DirectionsRoute>);
  }
```

Reply:
I was referring to https://github.com/mapbox/mapbox-navigation-android/blob/38ba88c80b847b37fae27590de9f600d3da28d0b/libnavui-maps/api/current.txt#L888-L891

Reply:
Ah, I see, fixed!